src/turnkeyml/build/export.py

```diff
@@ -172,19 +174,19 @@ def fire(self, state: build.State):
         shutil.copy(state.model, output_path)
 
         tensor_helpers.save_inputs(
-            [state.inputs], state.original_inputs_file, downcast=False
+            [state.inputs],
+            original_inputs_file(state.cache_dir, state.build_name),
+            downcast=False,
         )
 
         # Check the if the base mode has been exported successfully
         success_msg = "\tSuccess receiving ONNX Model"
         fail_msg = "\tFailed receiving ONNX Model"
 
         if check_model(output_path, success_msg, fail_msg):
-            state.intermediate_results = [output_path]
+            state.intermediate_results = output_path
 
-            stats = fs.Stats(
-                state.cache_dir, state.config.build_name, state.evaluation_id
-            )
+            stats = fs.Stats(state.cache_dir, state.build_name, state.evaluation_id)
```

Comment:
[no action needed for this PR]
Looks like a lot of the changes were simply caused by the auto formatter.
To avoid this we should consider a way to standardize this across all devs (or add a formatter step with a fixed config as part of git commit).

Reply:
Its actually that the line got shorter when `state.config.build_name` became `state.build_name`, that's the real change here.